godot-core/src/builtin/dictionary.rs

```diff
@@ -308,6 +308,19 @@ impl fmt::Debug for Dictionary {
         write!(f, "{:?}", self.to_variant().stringify())
     }
 }
+impl fmt::Display for Dictionary {
+    /// Formats `Dictionary` to match Godot's string representation
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{{ ")?;
+        for (count, entry) in self.iter_shared().enumerate() {
+            if count != 0 {
+                write!(f, ", ")?;
+            }
+            write!(f, "{}: {}", entry.0, entry.1)?;
+        }
+        write!(f, " }}")
+    }
+}
```

Comment:
Are the spaces after `{ ` and before ` }` consistent with the way Godot formats dictionaries?

Reply:
Yes, Its quite strange but it is how an empty dictionary is logged out.

Reply:
Particularly strange because arrays don't do that.